gse-copy-paste-afs/pom.xml

```diff
@@ -18,6 +18,10 @@
             <groupId>com.powsybl</groupId>
             <artifactId>powsybl-afs-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.powsybl</groupId>
```

Comment:
Is it the right place for this runtime dependency? We often add it with runtime scope and in an application module (something with a main)

Reply:
indeed (I needed it (as a provided dep) in a previous iteration). will be removed